libc/src/__support/FPUtil/FPBits.h

```diff
@@ -136,6 +136,11 @@ struct FPRepBase : public internal::FPLayout<fp_type> {
   LIBC_INLINE static constexpr StorageType bit_at(int position) {
     return StorageType(1) << position;
   }
+  LIBC_INLINE static constexpr StorageType merge(StorageType a, StorageType b,
```

Comment:
Please add a comment, it's not obvious what the naming means, and I don't think it's standard terminology.